store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/util/FederatedStoreUtil.java

```diff
@@ -282,7 +282,12 @@ public static String getDeprecatedGraphIds(final Operation operation) throws Gaf
 
     @Deprecated
     public static FederatedOperation<Void, Iterable<Schema>> getFederatedWrappedSchema() {
-        return new FederatedOperation.Builder().<Void, Iterable<Schema>>op(new GetSchema()).build();
+        return getFederatedWrappedSchema(true);
+    }
+    public static FederatedOperation<Void, Iterable<Schema>> getFederatedWrappedSchema(final boolean getOptimisedSchema) {
+        final GetSchema.Builder getSchema = new GetSchema.Builder();
+        getSchema.option("UnsupportedGetOptimisedSchema", String.valueOf(getOptimisedSchema));
+        return new FederatedOperation.Builder().<Void, Iterable<Schema>>op(getSchema.build()).build();
```

Comment:
The option given doesn't exist. If you want the optimised version you use `GetSchema.Builder().compact(true)`. Maybe this can be:
```suggestion
    public static FederatedOperation<Void, Iterable<Schema>> getFederatedWrappedSchema(final boolean getCompactSchema) {
        final GetSchema.Builder getSchema = new GetSchema.Builder();
        getSchema.compact(getCompactSchema);
        return new FederatedOperation.Builder().<Void, Iterable<Schema>>op(getSchema.build()).build();
```

Reply:
I would say this should be false by default

Reply:
It should yes. It might also be worth looking at what happens to non-wrapped `GetSchema` operations with the compact option set when they are sent to a federated store. I'm not sure that this option is preserved at the moment. More tests would be needed for this.